tests/framework/installer/cassandra_manifests.go

```diff
@@ -258,6 +273,7 @@ spec:
     racks:
       - name: "us-east-1a"
         members: %[2]d
+        configMapName: "cassandra-config-map"
```

Comment:
Please add this line commented out with a comment to the `cluster.yaml` example manifest in the `cluster/` dir and the example in the documentation https://rook.io/docs/rook/v1.3/cassandra-cluster-crd.html#sample.
In addition to that the option should be added to the list on the https://rook.io/docs/rook/v1.3/cassandra-cluster-crd.html#settings-explanation